chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urluptime/FixDatadogUrlUptimeAssertionsJob.java

```diff
@@ -0,0 +1,146 @@
+package gov.healthit.chpl.scheduler.job.urluptime;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+import com.datadog.api.client.ApiException;
+import com.datadog.api.client.v1.model.SyntheticsAPITest;
+import com.datadog.api.client.v1.model.SyntheticsAPITestConfig;
+import com.datadog.api.client.v1.model.SyntheticsAPITestType;
+import com.datadog.api.client.v1.model.SyntheticsAssertion;
+import com.datadog.api.client.v1.model.SyntheticsAssertionOperator;
+import com.datadog.api.client.v1.model.SyntheticsAssertionTarget;
+import com.datadog.api.client.v1.model.SyntheticsAssertionType;
+import com.datadog.api.client.v1.model.SyntheticsTestOptions;
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsHTTPVersion;
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsScheduling;
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsSchedulingTimeframe;
+import com.datadog.api.client.v1.model.SyntheticsTestRequest;
+
+import gov.healthit.chpl.scheduler.job.QuartzJob;
+import gov.healthit.chpl.util.DateUtil;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class FixDatadogUrlUptimeAssertionsJob extends QuartzJob {
+
+    private static final Integer HTTP_STATUS_OK = 200;
+    private static final String HTTP_METHOD_GET = "GET";
+    private static final Long SECONDS_IN_A_MINUTE = 60L;
+
+
+    @Autowired
+    private DatadogSyntheticsTestService datadogSyntheticsTestService;
+
+    @Value("${datadog.syntheticsTest.startTime}")
+    private String datadogTestStartTime;
+
+    @Value("${datadog.syntheticsTest.endTime}")
+    private String datadogTestEndTime;
+
+    @Value("${datadog.syntheticsTest.checkEveryMinutes}")
+    private Long datadogCheckEveryMinutes;
+
+    @Value("${datadog.syntheticsTest.timeout}")
+    private Integer datadogTestTimeout;
+
+    @Value("${datadog.syntheticsTest.location}")
+    private String datadogTestLocation;
+
+    @Override
+    public void execute(JobExecutionContext context) throws JobExecutionException {
+        SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
+        LOGGER.info("********* Starting the Fix Datadog Url Uptime Assertions Job *********");
+
+        datadogSyntheticsTestService.getAllSyntheticsTests().forEach(test -> {
+            Optional<SyntheticsAssertion> contentHeaderAssertion = getContentHeaderAssertionExist(test.getConfig().getAssertions());
+
+            if (contentHeaderAssertion.isPresent()) {
```

Comment:
Do we want a logging statement in an `else` condition of this check to indicate that the datadog test _is not_ being updated?